rainier-core/src/main/scala/com/stripe/rainier/core/Continuous.scala

```diff
@@ -211,3 +209,36 @@ object Uniform {
   def apply(from: Real, to: Real): Continuous =
     standard.scale(to - from).translate(from)
 }
+
+case class ContinuousMixture(components: Map[Continuous, Real])
```

Comment:
I still think we should just rename this to `Mixture` and delete the other one.

Reply:
Done

Reply:
Actually before I do... I'm just worried about removing the ability to have `Mixture`s for non-`Continuous` distributions. At the moment, `Categorical.toMixture` can work with any type of distribution.